templates/index.html

```diff
@@ -1,17 +1,172 @@
 {% extends 'base.html' %}
+{% load i18n %}
+{% get_current_language as LANGUAGE_CODE %}
+{% get_available_languages as LANGUAGES %}
+{% get_current_language_bidi as LANGUAGE_BIDI %}
 
-{% block title %}Home{% endblock %}
+{% block title %} {% trans "Home" %} {% endblock %}
 {% block content %}
-{% if user.is_authenticated %}
-    Hi {{ user.username }}!</br>
-    <a href="{% url 'user_view' user.pk %}">User information about current logged user</a></br>
-    <a href="{% url 'logout' %}">Logout here</a>
-{% else %}
-    <p>You are not logged in</p>
-    <a href="{% url 'login' %}">Login here</a>
-{% endif %}
-    <p>Links on this site:</p>
-    <a href="{% url 'polls:index' %}">Polls App</a></br>
-    <a href="{% url 'admin:index' %}">Admin Site</a></br>
-    <a href="{% url 'secret' %}">Secret (requires login to proceed)</a>
+    <div class="d-flex flex-column flex-md-row align-items-center p-3 px-md-4 mb-3 bg-dark border-bottom box-shadow">
+        <h5 class="my-0 mr-md-auto font-weight-normal"><a href="{% url 'home' %}">Django Project</a></h5>
+        <nav class="my-2 my-md-0 mr-md-3">
+            <a class="p-2 text-white" href="{% url 'polls:index' %}">{% trans "Polls App" %}</a>
+            <a class="p-2 text-white" href="{% url 'admin:index' %}">{% trans "Admin Site" %}</a>
+            <a class="p-2 text-white" href="{% url 'secret' %}">{% trans "Secret (requires login to proceed)" %}</a>
+            {% if user.is_authenticated %}
+                <a class="p-2 text-white" href="{% url 'user_view' user.pk %}">{% trans "Hi" %} {{ user.username }}!</a>
+            {% else %}
+                <a class="p-2 text-white" href="{% url 'signup' %}">{% trans "Register here" %}</a>
+            {% endif %}
+        </nav>
+        {% if user.is_authenticated %}
+            <a class="btn btn-outline-primary" href="{% url 'logout' %}">{% trans "Sign out" %}</a></br>
+        {% else %}
+            <a class="btn btn-outline-primary" href="{% url 'login' %}">{% trans "Sign In" %}</a></br>
+        {% endif %}
+    </div>
+    <div class="album py-5 bg-light">
```

Comment:
to samo z tym, przenieść to do base.html, robisz tak
```
<body>
<div class="album py-5 bg-light">
    <div class="container">
        {% block content %}
        {% endblock %}
    </div>
   </div>
</body>
```

a w tym pliku już po prostu dodajesz kontent do bloku 'content',, czyli
```
{% block content %}
<div class="row">
       <div class="col-md-4">
       i t.d.
       </div>
</div>
{% endblock %}
```